frontend/src/metabase/visualizations/echarts/cartesian/option/series.ts

```diff
@@ -44,6 +44,14 @@ import {
 import type { ChartMeasurements } from "../chart-measurements/types";
 import { getSeriesYAxisIndex } from "./utils";
 
+const getBlurLabelStyle = (
+  settings: ComputedVisualizationSettings,
+  hasMultipleSeries: boolean,
+) => ({
+  show: settings["graph.show_values"] && !hasMultipleSeries,
+  opacity: hasMultipleSeries ? CHART_STYLE.opacity.blur : 1,
```

Comment:
```suggestion
  show: settings["graph.show_values"],
  opacity: hasMultipleSeries ? CHART_STYLE.opacity.blur : 1,
```

Since we will now be using an opacity for when we have multiple series, we should remove that `!hasMultipleSeries` check from the `show` property. Otherwise, we would never see the other series labels on hover. If that were the case, opacity would not even matter.


https://github.com/metabase/metabase/assets/22608765/645dcb7a-39d6-439a-b28a-f569c4d59b5e

Reply:
Thanks, I simplified it in a similar way:
```
show: settings["graph.show_values"] && !hasMultipleSeries,
opacity: 1,
```

That is because hiding them entirely looks cleaner than blurring:
<img width="449" alt="Screenshot 2024-03-29 at 6 20 17 PM" src="https://github.com/metabase/metabase/assets/14301985/e3881149-65fb-4495-8e89-ba5f08b8565a">

<img width="452" alt="Screenshot 2024-03-29 at 6 20 36 PM" src="https://github.com/metabase/metabase/assets/14301985/1d1bc806-d07c-49bc-890a-a8d6c840cd7f">